microsoft-edge/progressive-web-apps-chromium/how-to/index.md

```diff
@@ -208,13 +379,13 @@ To add a service worker in your project:
 
 To confirm that your service worker is running:
 
-1.  Go to your web app at `http://localhost:8080`.
+1.  Browse to `http://localhost:8080` in Microsoft Edge and press `F12` to open DevTools.
```

Comment:
```suggestion
1. In Microsoft Edge, go to `http://localhost:8080` and then press `F12` to open DevTools.
```
https://learn.microsoft.com/en-us/style-guide/procedures-instructions/describing-interactions-with-ui

md template for f12: 
```
Or, press `Ctrl`+`Shift`+`I` (Windows, Linux) or `Command`+`Option`+`I` (macOS).
```